build.sbt

```diff
@@ -0,0 +1,18 @@
+enablePlugins(TypelevelCiReleasePlugin)
```

Comment:
I can't remember, is it necessary to manually enable this? I see you made the plugin self-hosting which is awesome, also serves as a basic test.

Reply:
`tlRelease` seems to be available even without it. So I guess it's not needed. Will have to remove from sbt-http4s-org plugin as well then